app/models/apple/episode.rb

```diff
@@ -382,9 +386,7 @@ def drafting?
     end
 
     def apple_upload_complete?
-      pdfs = feeder_episode.apple_podcast_delivery_files
-
-      pdfs.present? && pdfs.to_a.flatten.all?(&:apple_complete?)
+      feeder_episode.apple_podcast_container.skip_delivery?
```

Comment:
Bit odd conceptually (the upload is complete when we're skipping the delivery?)  Though I guess it's all the same - we skip the delivery when the upload is complete.  Same check.